packages/web-components/src/components/va-textarea/va-textarea.css

```diff
@@ -0,0 +1,26 @@
+@import '../../mixins/focusable.css';
+@import '../va-text-input/va-text-input.css';
+
+:host {
+  display: block;
+}
+
+:host([error]) label {
+  margin-top: 0;
+}
+
+label {
+  display: block;
+  margin-top: 2.8rem;
+  max-width: 48rem;
+}
+
+textarea {
+  display: block;
+  min-height: 5.2rem;
+  height: 16rem;
+  max-width: 48rem;
+  width: 100%;
+  padding: 1.2rem;
+  margin: 0.4rem 0;
+}
```

Comment:
Some of these values are slightly different than the React `<TextArea>` because I wanted to align them more closely to our 8px grid.